src/Sylius/Bundle/CoreBundle/Test/Services/SecurityService.php

```diff
@@ -45,14 +49,23 @@ public function __construct(
     /**
      * {@inheritdoc}
      */
-    public function logIn($email, $providerKey, Session $minkSession)
+    public function logIn($email, $providerKey = self::DEFAULT_PROVIDER_KEY, Session $minkSession)
```

Comment:
In fact, at this moment is not needed as the token is already authenticated. We have other setting hardcoded, `_security_user` has a `user` as a context name used in our firewalls, which is processed by `ContextListener`. If provider key is anything but not null, everything goes all right.